src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -135,7 +135,13 @@
 
 {% block form_widget_simple -%}
     {% if type is not defined or type != 'hidden' %}
-        {%- set attr = attr|merge({class: (attr.class|default('') ~ (type|default('') == 'file' ? ' custom-file-input' : ' form-control'))|trim}) -%}
+        {% set className = ' form-control' %}
+        {% if type|default('') == 'file' %}
+            {% set className = ' custom-file-input' %}
+        {% elseif type|default('') == 'range' %}
+            {% set className = ' form-control-range' %}
+        {% endif %}
+        {%- set attr = attr|merge({class: (attr.class|default('') ~ className)|trim}) -%}
```

Comment:
Change the conditional to one line with ternary operator
```twig
        {%- set attr = attr|merge({class: (attr.class|default('') ~ (type|default('') == 'file' ? ' custom-file-input' : (type|default('') == 'range' ? ' form-control-range' : ' form-control')))|trim}) -%}
```
Or apply the `-` operator to trim spaces and newlines
```twig
        {%- set className = ' form-control' -%}
        {%- if type|default('') == 'file' -%}
            {%- set className = ' custom-file-input' -%}
        {%- elseif type|default('') == 'range' -%}
            {%- set className = ' form-control-range' -%}
        {%- endif -%}
```

Reply:
Thank you